skuba-update/setup.py

```diff
@@ -14,47 +14,15 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import os
-
 from setuptools import setup
 
 
-def version():
-    """Return the version. Pass when VERSION file not found"""
-    pwd = os.path.dirname(os.path.abspath(__file__))
-    try:
-        with open(os.path.join(pwd, '..', 'VERSION')) as version_file:
-            return version_file.read().strip()
-    except FileNotFoundError:
-        return ''
-
-
 setup(
-    name="skuba-update",
-    version=version(),
-    author="SUSE Containers Team",
-    author_email="containers@suse.com",
-    description="Utility to automatically refresh and update a SUSE CaaSP cluster nodes",
-    long_description="Wraps zypper to refresh repositories and apply non interactive patches, including patches requiring reboot.",
-    license="Apache License 2.0",
-    keywords="CaaSP",
-    url="https://github.com/SUSE/skuba-update",
-    packages=['skuba_update'],
-    classifiers=[
-        'Intended Audience :: Developers',
-        'License :: OSI Approved :: Apache License 2.0',
-        'Operating System :: POSIX :: Linux',
-    ], data_files=[
+    data_files=[
```

Comment:
Why not put this in setup.cfg too? https://setuptools.readthedocs.io/en/latest/setuptools.html#configuring-setup-using-setup-cfg-files

Reply:
It can, but by default the - are replaced with underscore IIRC, when setup.cfg is used. I didn't want to deal with the migration of those files.

The best would indeed move to setup.cfg for everything, or better, pyproject.toml.

However I am not sure our packaging is ready for this.